code/datums/quirks/positive_quirks.dm

```diff
@@ -265,6 +298,17 @@
 /datum/quirk/item_quirk/tagger/add_unique(client/client_source)
 	give_item_to_holder(/obj/item/toy/crayon/spraycan, list(LOCATION_BACKPACK = ITEM_SLOT_BACKPACK, LOCATION_HANDS = ITEM_SLOT_HANDS))
 
+/datum/quirk/throwingarm
+	name = "Throwing Arm"
+	desc = "Your arms have a lot of heft to them! Objects that you throw just always seem to fly further than everyone elses, and you never miss a toss."
+	icon = "baseball"
+	value = 7
+	mob_trait = TRAIT_THROWINGARM
+	gain_text = "<span class='notice'>Your arms are full of energy!</span>"
+	lose_text = "<span class='danger'>Your arms ache a bit.</span>"
```

Comment:
```suggestion
	gain_text = span_notice("Your arms are full of energy!")
	lose_text = span_danger("Your arms ache a bit.")
```

Using the span macros look better.

Reply:
you think its worth to do a separate pr that makes all of the other quirk notices like this into macros?

Reply:
yes